### ISSUE TITLE
exchangeratesapi.io now requires an access key

### ISSUE THREAD

**Issue by mickecamino** [yawnsde/MMM-CurrencyExchange#8]
Discovered today when accessing exchangeratesapi.io:
`
success | false
-- | --
error |  
code | 101
type | "missing_access_key"
info | "You have not supplied an API Access Key. [Required format: access_key=YOUR_ACCESS_KEY]"
`

**Comment by fritzhildebrand** [comment]
Yes, this is very recent. Even if you get an access key, the free version by default only allows EUR as the base, and updates I think are limited to 250 per month. I have made some changes which makes it work for me. Happy to share as long as you are aware I am not a javascript expert. It also cuts the number of decimal places shown down to 4.

**Comment by mickecamino** [comment]
I signed up and found, just like you, that base is default at EUR and 250 requests per month, its like 8 requests per day.
Please share the changes you have made. I have some ideas to workaround the EUR base and get correct quotes for other currencies.

**Comment by fritzhildebrand** [comment]
Making the attached changes to the config.js and MMM-CurrencyExchange.js worked for me. It gets the exchange rates against EUR for the currencies and then recalculates the rate against your entered base currency (USD in the example). Also amended the apiBase from https to http, changed the updateInterval to once every 3 hours (max of 248 times per month), and changed the number of decimal place to 4.
[exchange - changes.txt](https://github.com/yawnsde/MMM-CurrencyExchange/files/6274354/exchange.-.changes.txt)

**Comment by BrielC** [comment]
Thanks Fritz for sharing your findings and solution. My module is running again. Much appreciated

**Comment by hermannw** [comment]
Thank you Fritz for making this work again!
Any idea how I can change the font-size? I tried in custom.css:
```
 .MMM-CurrencyExchange {
    font-size: 150%; //went all the way up to 500%
}

This works on other modules but not here...

Hermann

**Comment by mickecamino** [comment]
It's because it's hard coded in the .js file.
